tests/Microsoft.Bot.Builder.Dialogs.Tests/WaterfallTests.cs

```diff
@@ -466,6 +466,11 @@ public void TrackDependency(string dependencyTypeName, string target, string dep
                 throw new NotImplementedException();
             }
 
+            public void TrackPageView(string dialogName, IDictionary<string, string> properties = null, IDictionary<string, double> metrics = null)
```

Comment:
and this isn't a realistic scenario right, just testing? we are suggesting if you _don't_ implement TrackPageView then you _don't_ implement IBotPageViewTelemetryClient

Reply:
Agreed. I will remove.